doc/source/code_editors.rst

```diff
@@ -21,43 +49,25 @@ IDE and Editor Recommendations
   	Plugin Available at: https://github.com/steenhulthin/reStructuredText_NPP
   	For Better creation and reformation of tables in RST using Vim use the following Plugin 
   	https://github.com/nvie/vim-rst-tables/
-
+	
+	
 -Nano
 	Next Editor is also targeted at more experienced users that aren´t interested in nice UI Designs. 
 	But I wouldn´t recommend using Nano because there are no RST Plugins for Nano
-
--Atom
+	
+	-Atom
 	Atom greets you with the promise being “a hackable text editor for the 21st Century” 
 	Customization and Packages can be installed trough the integrated App store. 
 	To use RST in Atom you can install “rst-preview” Note: The plugin only can be used as previewer of RST
 	https://atom.io/packages/rst-preview
 	Additionally, for RST language support use : https://atom.io/packages/language-restructuredtext
 	And furthermore, use https://github.com/textmate/restructuredtext.tmbundle
-
--Eclipse
-
-	Eclipse was originally developed for Java but with Plugins it can be modified to
-	run man programming languages as well as RST
-	https://marketplace.eclipse.org/content/liclipsetext
-
+	
 -Sublime
 
 	Sublime is a proprietary Text editor fpr Windows, Mac and Linux. 
 	It supports Many useful future proof features like ARM64 or GPU rendering
 	RST Support is built right in and the programming language can be changed in the lower right hand corner.  
-
--Intellij
-
-	Intellij is a IDE for Java programming 
-	Intellij has so called deep intelligence by giving relevant suggestions in every context by analyzing your source code
-	In addition the IDE has a good out-of-the-box experience 
-  	Although, RST hast to be installed at https://plugins.jetbrains.com/plugin/7124-restructuredtext/versions
-
-
--Xcode
-
-	Xcode is Apple´s IDE used for development of **macOS, iOS, iPadOS and tvOS software**.
-	Therefore, other IDEs would be recommended
 	
 -VSCode
```

Comment:
VS is clearly IDE. Otherwise good

Reply:
VS code is kind of a hybrid, but I get your point.
I will change it accordingly
Thanks for your input